controllers/ovncontroller_controller.go

```diff
@@ -495,7 +494,17 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 	}
 
 	// Create or Update additional Physical Network Attachments
-	networkAttachments, err := ovncontroller.CreateOrUpdateAdditionalNetworks(ctx, helper, instance, ovsServiceLabels)
+	// network to attach to
+	networkAttachments := []string{}
+	networkAttachmentsNoPhysNet := []string{}
+	if instance.Spec.NetworkAttachment != "" {
+		networkAttachments = append(networkAttachments, instance.Spec.NetworkAttachment)
+		networkAttachmentsNoPhysNet = append(networkAttachmentsNoPhysNet, instance.Spec.NetworkAttachment)
+	}
+
+	networkAttachmentsToAdd, err := ovncontroller.CreateOrUpdateAdditionalNetworks(ctx, helper, instance, ovsServiceLabels, networkAttachments)
+	networkAttachments = append(networkAttachments, networkAttachmentsToAdd...)
```

Comment:
This changing the order of interfaces so during update to this version of operator ovs pods will be restarted, which we should avoid.

Reply:
Yes order is changed, for instance networkAttachment=tenant and nicMappings=datacentre:enp8s0
Before this patch nic order was datacentre, tenant
And now with this patch order is tenant, datacentre
So if you can just reorder(networkAttachmentsToAdd + networkAttachments) the append in L506 it should be enough to persist original order.

Reply:
Done!